### PR TITLE
Bring in the UI work

### PR DESCRIPTION
**Pull request by kerneltime**
PR for UI work from @marksoper.
To build the UI code run
```
 INCLUDE_UI=true make
```

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "repository" : {
+    "type" : "git",
+    "url" : "https://github.com/vmware/docker-volume-vsphere.git"
+  },
+  "author": {
+    "name": "Mark Soper",
```
Comment (pdhamdhere) [inline ui/package.json]: We generally avoid individual's name in code/repository. I would suggest to replace with following;
name = SABU, VMware
email = cna-storage@vmware.com
url = https://github.com/vmware/docker-volume-vsphere

File: ui/scripts/controllers/dvol-main.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    DvolTenantService, DvolTenantGridService, DvolVsanService) {
+
+    // -----------------------------
+    //
+    // Temporary
+    //
+    // Here when the app is loaded we're temporarily making
+    // a call to DvolVsanService to test connectivity
+    //
+    // TODO: remove this
```
Comment (pdhamdhere) [inline ui/scripts/controllers/dvol-main.js]: I assume this will be removed at later point when UI is fully integrated with backend.

Comment (marksoper) [inline ui/scripts/controllers/dvol-main.js]: yes

**Comment by kerneltime** [comment]
Merging this change in. From here on, UI changes should be incremental small PRs.

**Comment by kerneltime** [comment]
Fixes https://github.com/vmware/docker-volume-vsphere/issues/637

**Pull request merged**